novawallet/Modules/Onboarding/AccountConfirm/AccountConfirmViewFactory.swift

```diff
@@ -73,53 +73,61 @@ final class AccountConfirmViewFactory: AccountConfirmViewFactoryProtocol {
         for interactor: BaseAccountConfirmInteractor,
         wireframe: AccountConfirmWireframeProtocol
     ) -> AccountConfirmViewProtocol? {
-        let view = AccountConfirmViewController(nib: R.nib.accountConfirmViewController)
+        let localizationManager = LocalizationManager.shared
+        let mnemonicViewModelFactory = MnemonicViewModelFactory(localizationManager: localizationManager)
+
+        let presenter = AccountConfirmPresenter(
+            wireframe: wireframe,
+            interactor: interactor,
+            mnemonicViewModelFactory: mnemonicViewModelFactory,
+            localizationManager: localizationManager
+        )
 
-        #if F_DEV
-            view.skipButtonTitle = LocalizableResource { locale in
-                R.string.localizable.confirmationSkipAction(preferredLanguages: locale.rLanguages)
-            }
-        #endif
+        let view = AccountConfirmViewController(
+            presenter: presenter,
+            localizationManager: localizationManager
+        )
 
-        let presenter = AccountConfirmPresenter()
+//        #if F_DEV
```

Comment:
Let's remove if we don't need this. But I don't see where we moved this button logic

Reply:
logic for skip added for F_DEV compilation flag